### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by MrSinisterX** [ovh/venom#714] [General]
Hello, is it possible to get a new release (pre-release) please? There is a very important feature - [ShouldJSONEqual](https://github.com/ovh/venom/pull/676) for me at least, that isn't yet released for some time.

Thank you!

**Comment by fokion** [comment]
I agree with the sentiment , there  is no communication from the core maintainers regarding releases and roadmap. PRs are open for more than a month without even a comment. If it is meant to be a OVH tool that got abandoned we should know about it in order to either move it to another organisation with more active maintenance.

**Comment by yesnault** [comment]
Hi @fokion @MrSinisterX  Venom is widely used and still maintained, there is no problem with that. Every PR is checked and we do not want to introduce bugs or breaking changes. It takes time, for sure, but we don't want to do things in a hurry. 
BTW, we are looking to do a beta.3 release this week.

**Reply by yesnault** [reply]
it's here https://github.com/ovh/venom/releases/tag/v1.2.0-beta.3
